app/ide-desktop/lib/dashboard/src/components/AriaComponents/Radio/RadioGroupContext.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file
+ *
+ * This file provides a context for RadioGroup and Radio components.
+ */
+
+import * as React from 'react'
+
+import invariant from 'tiny-invariant'
+
+import * as eventCallback from '#/hooks/eventCallbackHooks'
+
+/**
+ * Props for {@link RadioGroupContextProps}
+ */
+export interface RadioGroupContextProps {
+  readonly pressedRadio: string | null
+  readonly setRadioPressed: (value: string) => void
```

Comment:
`setRadioPressed` -> `setPressedRadio` for consistency.

but imo it would be better to call it `value`

Reply:
See: https://github.com/enso-org/enso/pull/10178#issuecomment-2149000713

Reply:
makes sense (but of course, it should still be renamed for consistency - either all `radioPressed` or all `pressedRadio`)